util/travis-deploy-staging.sh

```diff
@@ -25,6 +25,9 @@ if [[ "${APP_PATH}" == "webapp" ]]; then APP_DEPS="${APP_DEPS}|api|shared"; fi
 if [[ "${APP_PATH}" == "revisions/service" ]]; then APP_DEPS="${APP_DEPS}|revisions|shared"; fi
 APP_DEPS_REGEX="^(${APP_DEPS})/"
 
+EXCLUSIONS="_test.go$$"
+if [[ "${APP_PATH}" == "webapp" ]]; then EXCLUSIONS="${EXCLUSIONS}|webapp/components/test/"
```

Comment:
Do we need the `if`? Can we just always ignore these files?

Reply:
Fair point. Done.

Reply:
BTW, you're also missing a `fi` here, but I guess you're removing the whole `if` anyway.